packages/@aws-cdk/aws-cognito/test/integ.user-pool-idp.amazon.ts

```diff
@@ -7,7 +7,7 @@ import { ProviderAttribute, UserPool, UserPoolIdentityProviderAmazon } from '../
  * * If you plug in valid 'Login with Amazon' credentials, the federated log in should work.
  */
 const app = new App();
```

Comment:
missed this in this last iteration. I believe the corresponding expectation file - `integ.user-pool-idp.expected.json` - needs to be deleted.